Infrastructure-Scripts/Invoke-CdnContentPurge.ps1

```diff
@@ -32,7 +31,7 @@ Param(
     [String]$CDNProfileName,
     [Parameter(Mandatory = $true)]
     [String]$CDNEndPointName,
-    [Parameter(Mandatory = $false)]
+    [Parameter(Mandatory = $true)]
     [String]$PurgeContent = ""
```

Comment:
No longer needs the default to ""

Reply:
if ( $PurgeContent -eq "" ) { } needed now if it is now a mandatory parameter? Still needed as we don't want to run a purge content with a blank pattern match. Waste of resources.